style.css

```diff
@@ -0,0 +1,330 @@
+body {
+    margin: 0px;
+}
+
+.main-container {
+    height: 1000px;
+    background-color: rgb(173, 27, 27);
+    display: grid;
+    grid-template-areas:
+    "left main right" ;
+    grid-template-columns: 250px 1fr 250px;
+    gap: 3px;
+}
+
+.left {
+    grid-area: left;
+    background-color: aliceblue;
+}
+
+main {
+    grid-area: main;
+    background-color: aliceblue;
+    height: 1000px;
+}
+
+.right {
+    grid-area: right;
+    background-color: aliceblue;
+}
+
+/* Section for styling left side */
+.left > div {
+    display: inline-block;
+}
+
+.icons {
+    width: 50px;
+}
+
+.icons > img {
+    width: 50px;
+    height: 50px;
+    padding: 10px;
+}
+
+.buttons {
+    width: 80px;
+    position: relative;
+    bottom: 25px;
+    left: 10px;
+    & button {
+        display: block;
+        margin: 10px;
+        margin-bottom: 50px;
+    }
+    
+}
+
+#special {
+    margin-bottom: 0px;
+}
+
+button {
+    font-size: larger;
+    border: none;
+    background-color: aliceblue;
+}
+
+button:hover {
+    color: orange;
+    text-decoration: underline;
+}
+
+button:focus {
+    color: rgb(173, 27, 27);
+}
+
+.post {
+    width: 90%;
+    padding: 10px;
+    border-radius: 21px;
+    margin-left: 10px;
+    color:aliceblue;
```

Comment:
Try to keep the formatting clean and consistent - you're missing a whitespace between `color:` and `aliceblue`.